rust/foxglove/CONTRIBUTING.md

```diff
@@ -2,16 +2,27 @@
 
 ## Development
 
+### Dependencies
+
+- Rust, installed via [rustup](https://rustup.rs/)
+- The python environment described in the [Python SDK README](../python/foxglove-sdk/README.md)
+
 ### Generate Protobuf schemas
 
 ```bash
 cargo run --bin foxglove-proto-gen
 ```
 
+### Generate schemas for the Python SDK
+
+```bash
+yarn generate
```

Comment:
The changes to this file and comments about Python don't seem relevant to the rust `foxglove` package